R/filter_parameters.R

```diff
@@ -57,7 +57,8 @@ filter_parameters <- function(x, ..., parameters = NULL) {
   check_filter_dots(rlang::enquos(...))
   # check for type
   if (!inherits(x, "tune_results")) {
-    rlang::abort(paste0(cl_x, " should have class 'tune_results'."))
+    cli::cli_abort("{.arg x} should have class {.cls tune_results};
```

Comment:
```suggestion
    cli::cli_abort("{.arg {caller_arg(x)}} should have class {.cls tune_results};
```

Can also delete the line that creates `cl_x` above!